src/components/input-date-picker/input-date-picker.tsx

```diff
@@ -939,4 +944,14 @@ export class InputDatePicker
           )
           .join("")
       : "";
+
+  private parseNumerals = (value: string): string =>
```

Comment:
Sidebar: @benelan Is there a util from input or input-number that could be used here? If so, we should refactor in a separate PR.

Reply:
This isn't used in input or input number, the reason this is here is because trying to delocalize dates was causing issues since they aren't numbers due to the separators. So I ended up iterating over the characters and just localizing the numbers. It could be useful in input-time for the same reason so it may be worth pulling out into a util or even adding it as a method to the NumberStringFormat class

Reply:
To clarify, I added this method along with the formatNumerals one when I set up numbering system in the component but it got removed at some point

Reply:
Thanks for the background, sir! 🎩